src/statusBar.ts

```diff
@@ -63,7 +63,13 @@ class StatusBarImpl implements vscode.Disposable {
     let foreground;
     let background;
 
-    const colorToSet = configuration.statusBarColors[ModeName[mode].toLowerCase()];
+    let colorToSet = configuration.statusBarColors[ModeName[mode].toLowerCase()];
+
+    // If no color defined, fallback to normal mode color
+    if (colorToSet === undefined) {
+      colorToSet = configuration.statusBarColors[ModeName[ModeName.Normal].toLowerCase()];
```

Comment:
What is there's no normal mode color defined? Should we just log a debug message and return instead?

Reply:
The problem I have with that is the existing color will be left as active. If no normal is defined apparently it will be annoying bright red

Reply:
Where does the red come from? It's just strange that we are using the normal mode color as default as it may be null so maybe we should either:

1) validate the config and if they don't provide all the mode colors, we disable the status bar color.
2) if it's undefined, delete the `colorCustomizations` so we return back to vscode defaults.